### ISSUE TITLE
agent.jar无法删除

### ISSUE THREAD

**Issue by admln** [4ra1n/shell-analyzer#3]
看到代码里面进行`loadAgent`之后进行了`vm.detach();`，但是为什么不关掉web容器，就算gui.jar关掉，agent.jar也无法删除？
<img width="328" alt="image" src="https://github.com/4ra1n/shell-analyzer/assets/6093870/b13126be-af3a-49dd-a414-dffb97060b26">

**Comment by 4ra1n** [comment]
这是windows的问题，java层面处理不了
